markdown/3.0/en/documentation/upgrade-guide.md

```diff
@@ -0,0 +1,181 @@
+## Upgrade from version 1.2.x to 3.0.0
+We strongly recommend to upgrade your system to use the latest version of FACT-Finder Web Components. While we will do critical bug fixes for older versions, new features are likely to be implemented only in newer versions. In addition the newer versions stick to newer technology decreasing your loading time and improving internal speed. We promise to make each upgrade progress as ease as possible.
+
+In general you can keep track of our progress, changes and new features in the [release notes](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/CHANGELOG.md).
+
+There are only two major changes and some minor API changes to take care of when upgrading from version 1.2.x to version 3.0.0. In addition css mixins have been removed. You can use normal css selector instead. Further more `ffw-` is introduced as name prefix for FACT-Finder Web Components custom class names.
+
+### 2 major changes
+- Aligning with the current trend FACT-Finder Web Components are now shipped as ES6 Module instead of HTML Imports. Hence you have to load `bundle.js` instead of the HTML import as follows:
```

Comment:
```suggestion
- Aligning with the current trend FACT-Finder Web Components are now shipped as ES6 Module instead of HTML Imports. Which is why you have to load `bundle.js` instead of the previous HTML import.
```